modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -22,6 +22,7 @@ TIP: You can also use the *Git Repo URL ** field on the *Create Workspace* page
 
 * Your organization has a running instance of {prod-short}.
 * You know the FQDN URL of your organization's {prod-short} instance: `pass:c,a,q[{prod-url}]`.
+* Optional: xref:authenticating-yourself-to-a-git-server-from-a-workspace.adoc[Authentication to the Git server] is configured.
```

Comment:
Rewrite for parallelism. The rest of the sentences are in the present tense. So ensure you follow the guideline given below to rewrite the prerequisites accordingly.

Please refer to: https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

Reply:
@Srivaralakshmi, thank you. `Authentication to the Git server is configured` also appears to be in the present tense. What do you mean by `parallelism` on this line?

Reply:
The structure of the sentence. The rest starts with You and Your, 2nd person pronouns.

Reply:
Try something like: You have the authentication configured to the Git server.

Reply:
Thank you for the call. Rewriting it.